db/seeds.rb

```diff
@@ -6,12 +6,12 @@
 if Rails.env.development?
   # Create a few dummy accounts for use during development.  These accounts all
   # have 'password' as password and are setup for mobile OTP delivery.
-  mobile = 4_155_555_555
+  mobile = 0
   %w(test1@test.com test2@test.com).each do |email|
     User.find_or_create_by!(email: email) do |user|
       user.skip_confirmation!
       user.reset_password('password', 'password')
-      user.unconfirmed_mobile = mobile.to_s
+      user.unconfirmed_mobile = '+1 (415) 555-%04d' % mobile
       user.mobile_confirm
     end
     mobile += 1
```

Comment:
Setting the last digit of the phone number for each user can be achieved more easily with Ruby's [`#each_with_index`](http://ruby-doc.org/core-2.3.1/Enumerable.html#method-i-each_with_index).